lib/config/optionsconfiguration.go

```diff
@@ -184,15 +186,35 @@ func (opts OptionsConfiguration) StunServers() []string {
 	for _, addr := range opts.RawStunServers {
 		switch addr {
 		case "default":
-			defaultPrimaryAddresses := make([]string, len(DefaultPrimaryStunServers))
-			copy(defaultPrimaryAddresses, DefaultPrimaryStunServers)
-			rand.Shuffle(defaultPrimaryAddresses)
-			addresses = append(addresses, defaultPrimaryAddresses...)
+			_, records, err := net.LookupSRV("stun", "udp", "syncthing.net")
```

Comment:
This should still be configurable I think

Reply:
That might be a little awkward. At the moment the user is either going to roll with `default` aka "choose for me" or is providing his own addresses. If we make the SRV address configurable we'll end up with a mix.